### ISSUE TITLE
Snapping sensation with range of motion

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9874]
**Preferred term label:**
Snapping sensation with range of motion

**Synonyms**
Snapping, clicking, catching with range of motion

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Similar to HP:0012479 but more general

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Saggital band disruption

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Not sure I understand, could you provide the disease context?

**Comment by MickeySegal** [comment]
Seen with "Sagittal band disruption": https://www.orthobullets.com/hand/6029/sagittal-band-rupture "snapping sensation with extension".

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000298
- label: Snapping sensation at metacarpophalangeal joint
- parents: [Abnormal joint physiology (HP:0034430)](https://hpo.jax.org/app/browse/term/HP:0034430)
- pmid: 
- def: A feeling of clicking or snapping at the knuckle (metacarpophalangeal joint) upon extension of flexion of the finger.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9874
- synonyms:

**Comment by MickeySegal** [comment]
We may need a more general term.  

Also used for https://www.ncbi.nlm.nih.gov/books/NBK431067/ and https://www.ncbi.nlm.nih.gov/books/NBK470249/
